test/integration/targets/azure_rm_virtualnetwork/tasks/main.yml

```diff
@@ -4,6 +4,17 @@
     resource_group: "{{ resource_group }}"
     state: absent
 
+- name: Create virtual network
+  azure_rm_virtualnetwork:
+    name: my_test_network
+    address_prefixes_cidr:
+      - 10.1.0.0/16
+      - 172.100.0.0/16
+    tags:
+      testing: testing
+      delete: on-exit
+    resource_group: "{{ resource_group }}"
+
 - name: Create virtual network
```

Comment:
so you are running the same playbook twice to make sure it won't crash, right?

so here should be assert that state has changed, and below next task we should assert that state not changed, right?

or there's idempotency problem in this module?

Reply:
idempotent check is in the later of the file